frontend/yarn.lock

```diff
@@ -0,0 +1,5890 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
@MaxWofford I was under the impression we were not committing yarn.lock?

@zachlatta: I believe it was you who set this precedent, would you mind sharing the rationale?